isubit/iastate_theme
# D8 -1590 Host Academicons locally

Right now we pull FontAwesome and Academicons from a remote source. But that remote source is going away, so we are going to host it.

**To test:** 
1. Create a D8 faculty site and switch to this branch of the base theme.
2. Make sure you're seeing our little pencil and plus icons around the site.
3. Add social media icons to the footer and/or people profile, including google scholar or ResearchGate. Do the correct icons show up?

The focus is on the google scholar and ResearchGate icons since those are the ones we currently include.
